### PR TITLE
added company cariboulabs

### PR DESCRIPTION
**Pull request by AlonSpinner**
added our company, cariboulabs

### CODE REVIEW

**Comment by AlonSpinner** [comment]
added our company, cariboulabs

File: logos/cariboulabs-logo.png
Comment (mluggy) [inline logos/cariboulabs-logo.png:1]: logo should be absolute-aligned

File: companies/510057606.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Cariboulabs",
```
Comment (mluggy) [inline companies/510057606.json:2]: where did you pull the ח.פ from?

File: companies/510057606.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "description": "We develop software defined communication solutions, specializing in robotics applications",
+    "categoryId": 3,
+    "size": "xs",
+    "logoFile": "logos/cariboulabs-logo.png",
+    "websiteUrl": "https://github.com/cariboulabs",
```
Comment (mluggy) [inline companies/510057606.json:7]: no homepage?

**Review by mluggy — CHANGES_REQUESTED** [review]

File: companies/510057606.json (hunk 1 above)
Comment (mluggy) [reply companies/510057606.json:2]: Cariboulabs is [516525334](https://ica.justice.gov.il/GenericCorporarionInfo/SearchCorporation?unit=8#)

File: companies/510057606.json (hunk 2 above)
Comment (AlonSpinner) [reply companies/510057606.json]: fixed, thanks :)

File: companies/510057606.json (hunk 1 above)
Comment (AlonSpinner) [reply companies/510057606.json]: whoops. thank you!

File: logos/cariboulabs-logo.png
Comment (AlonSpinner) [reply logos/cariboulabs-logo.png:1]: Not exactly sure what you mean, but I centered it and made it a webp.

**Review by mluggy — APPROVED** [review]

**Pull request merged**